src/AppBundle/Entity/Report/Traits/StatusTrait.php

```diff
@@ -46,28 +60,43 @@ public function setStatusCached(array $status)
         $this->statusCached = json_encode($status);
     }
 
+    /**
+     * @return string
+     */
+    public function getReportStatusCached()
+    {
+        return $this->reportStatusCached;
+    }
+
 
     /**
-     * Update the status cache of the given sections
+     * Update the status cache of the given sections,
+     * and also the report.reportStatusCached using the cache
+     *
      * using the `ReportService::getSectionStateNotCached`
      *
      * @param array $sectionIds
      */
     public function updateSectionsStatusCache(array $sectionIds)
     {
         $currentStatus = $this->getStatusCached();
+        $statusObject = $this->getStatus();
```

Comment:
$currentSectionStatus = $this->getStatusCached();
$currentReportStatus = $this->getStatus();

Makes a lot more sense (if Ive understood correctly) :)

Reply:
yes, good idea. renamed